nodejs/eks/examples/migrate-nodegroups/echoserver.ts

```diff
@@ -0,0 +1,137 @@
+import * as eks from "@pulumi/eks";
+import * as k8s from "@pulumi/kubernetes";
+import * as pulumi from "@pulumi/pulumi";
+
+// Create the echoserver workload's Service, Deployment and Ingress.
+export function create(
+    name: string,
+    replicas: pulumi.Input<number>,
+    namespace: pulumi.Input<string>,
+    ingressClass: pulumi.Input<string>,
+    provider: k8s.Provider,
+): k8s.core.v1.Service {
+
+    const labels = {app: name};
+
+    // Create the Service.
+    const service = createService(name, labels, namespace, provider);
+    const serviceName = service.metadata.apply(m => m.name);
+
+    // Deploy the echoserver in the general, standard nodegroup.
+    const deployment = createDeployment(name, replicas, labels, namespace, provider);
+
+    // Create the Ingress.
+    const ingress = createIngress(name, labels, namespace, ingressClass,
+        serviceName, provider);
+
+    return service;
+}
+
+// Create the Ingress.
+export function createIngress(
+    name: string,
+    labels: pulumi.Input<any>,
+    namespace: pulumi.Input<string>,
+    ingressClass: pulumi.Input<string>,
+    serviceName: pulumi.Input<string>,
+    provider: k8s.Provider,
+): k8s.extensions.v1beta1.Ingress {
```

Comment:
```suggestion
): k8s.networking.v1beta1.Ingress {
```

Reply:
This new `networking` group + apiversion is only available on k8s >= 1.14. EKS only support upto 1.13 as of today, so skipping this suggestion.